src/RestControllers/RestControllerHelper.php

```diff
@@ -65,28 +65,50 @@ public static function validationHandler($validationResult)
      */
     public static function handleProcessingResult($processingResult, $successStatusCode, $isMultipleResultResponse = false)
     {
+        $fhirApi = self::is_fhir_request(self::getRequestEndPoint());
         $httpResponseBody = [
             "validationErrors" => [],
             "internalErrors" => [],
             "data" => []
         ];
 
-        if (!$processingResult->isValid()) {
-            http_response_code(400);
-            $httpResponseBody["validationErrors"] = $processingResult->getValidationMessages();
-        } else if ($processingResult->hasInternalErrors()) {
-            http_response_code(500);
-            $httpResponseBody["internalErrors"] = $processingResult->getInternalErrors();
-        } else {
-            http_response_code($successStatusCode);
-            $dataResult = $processingResult->getData();
+        if ($fhirApi === true) {
+            $httpResponseBody = [];
+            if (!$processingResult->isValid()) {
+                http_response_code(400);
+                $httpResponseBody["validationErrors"] = $processingResult->getValidationMessages();
+            } elseif ($processingResult->hasInternalErrors()) {
+                http_response_code(500);
+                $httpResponseBody["internalErrors"] = $processingResult->getInternalErrors();
+            } else {
+                http_response_code($successStatusCode);
+                $dataResult = $processingResult->getData();
```

Comment:
For now until I get Outcomes done which i'll need to refactor our Fhir apis to handle outcomes in resource. (need uris etc)